atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -47,6 +52,76 @@ def __init__(self, tasker, workflow, registries, group=True, goarch=None):
         self.registries = registries
         self.worker_registries = {}
 
+    def submit_manifest_list(self, registry, registry_conf, manifest_list_spec):
+        docker_secret_path = registry_conf.get('secret', None)
+        with NamedTemporaryFile(prefix='manifest-list', suffix=".yml", mode='w') as fp:
+            yaml.dump(manifest_list_spec, stream=fp)
+            fp.flush()
+            self.log.debug("Wrote to file %s with config %s", fp.name, docker_secret_path)
+
+            # --docker-cfg may be rendundant here, but it's how the tool should work
+            cmd = ['manifest-tool', '--docker-cfg=%s' % docker_secret_path,
+                   'push', 'from-spec', fp.name]
+            # docker always looks in $HOME for the .dockercfg, so set $HOME to the path
+            # subprocess.check_call(cmd, env={'HOME': docker_secret_path})
+            try:
+                subprocess.check_call(cmd)
+            except CalledProcessError as exc:
+                self.log.error("manifest-tool failed with %s", exc.output)
```

Comment:
`check_output(cmd)` must be used to have `output` in exception, `check_call` won't cut it

Reply:
Fixed.